modules/auth/repo_form.go

```diff
@@ -450,6 +450,7 @@ type MergePullRequestForm struct {
 	MergeTitleField   string
 	MergeMessageField string
 	ForceMerge        *bool `json:"force_merge,omitempty"`
+	MergeWhenChecksSucceed bool
```

Comment:
pleace exec `make fmt` and commit result

Reply:
Done.